mods/ts/rules/gdi-structures.yaml

```diff
@@ -760,3 +760,58 @@ GAPLUG4:
 		Type: plug.droppod
 	Power:
 		Amount: -20
+
+GAFIRE:
+	Inherits: ^Building
+	Inherits@IDISABLE: ^DisableOnLowPowerOrPowerDown
+	Inherits@SHAPE: ^3x2Shape
+	Buildable:
+		BuildPaletteOrder: 190
+		Prerequisites: proc, gatech, ~structures.gdi, ~techlevel.superweapons
+		Queue: Building
+		Description: actor-gafire.description
+	Valued:
+		Cost: 1500
+	Tooltip:
+		Name: actor-gafire.name
+	Building:
+		Footprint: xxx xxx
+		Dimensions: 3,2
+	Health:
+		HP: 100000
+	WithIdleOverlay@lights:
+		RequiresCondition: !build-incomplete
+		Sequence: idle-lights
+		Palette: player-nobright
+		IsPlayerPalette: true
+	WithIdleOverlay@lights-bright:
+		RequiresCondition: !build-incomplete
```

Comment:
could we pause lights on being on low power / disabled?

Reply:
Fixed